flask_app/models/user.py

```diff
@@ -1,7 +1,95 @@
-import re
 from flask_app.config.mysqlconnection import connectToMySQL
-from flask_bcrypt import Bcrypt
 from flask_app import app
-from flask import flash
+from flask_app.models import player
+from flask_bcrypt import Bcrypt        
 bcrypt = Bcrypt(app)
+
+import re
+from flask import flash
+
+
 EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+
+class User:
+    def __init__( self , data ):
+        self.id = data['id']
+        self.firstname = data['firstname']
```

Comment:
let's change these to first_name and last_name to match with created_at and updated_at